HW03/ihar_shkuryn/spec/homework3_task_01_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative '../ihar_shkuryn_hw_03_t_01'
+
+RSpec.describe Homework3 do
+  describe 'Unnormal log' do
+    let(:obj) { described_class.new }
+
+    context 'when log is a nil' do
+      let(:log) { nil }
+
+      it 'output is a empty' do
+        expect(obj.task1(log)).to eq('')
+      end
+    end
+
+    context 'when got array in params' do
+      let(:log) { ['error']  }
```

Comment:
consider adding all the logs you have been given. you can assign them in spec or use fixtures